### DISCUSSION TITLE
Routing colocation / Fractals structure

### DISCUSSION THREAD

**Discussion by theguriev** [nuxt/nuxt#23516] [Questions]
Hey, guys.

Is there any easy way to organize components, pages and composable in a fractal structure? I would like something similar to how it works in Nextjs https://nextjs.org/docs/app/building-your-application/routing/colocation

**Comment by dbzx10299** [comment]
Yes, take a look at this https://geistjs.com/blog/advanced-file-structure-and-colocation-in-nuxt
